### DISCUSSION TITLE
What happened to 'Allow assertJsonStructure in strict mode on TestResponse'

### DISCUSSION THREAD

**Discussion by p487morgan** [laravel/framework#46504] [Q&A]
I'm trying to understand what happened to this archived idea https://github.com/laravel/ideas/issues/820

It says 
> [taylorotwell](https://github.com/taylorotwell) closed this as [completed](https://github.com/laravel/ideas/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) [on Mar 13, 2018](https://github.com/laravel/ideas/issues/820#event-1519481110)

but I can't find it.

Other people are looking for the same functionality https://laracasts.com/discuss/channels/testing/find-extra-elements-with-assertjsonstructure

Any ideas?

**Comment by henzeb** [comment]
How about: assertExactJson?

**Reply by p487morgan** [reply]
That is great if you know the data (e.g. keys and values), but I'm looking for the equivalent method when you only know the structure (only keys)
